bench/run_benchmarks.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+import time
+import argparse
+import sys
+import os.path
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'lib'))
+
+def sfilter(suite, key):
+    return suite is None or key.startswith(suite)
+
+def bench_all(suite, just_list, **kw):
+    for bench in benchmarks.BENCHMARKS:
+        if sfilter(suite, bench.bench_prefix):
+            if just_list:
+                print(bench.bench_prefix, "-", bench.description)
+            else:
+                bench.run(**kw)
+
+def bench(args):
+    if args.cputime:
+        clock = time.clock
+    else:
+        clock = time.time
+    bench_all(
+        timer=clock, verbose=args.verbose, series=args.series, runtime=args.runtime,
+        suite=args.bench_suite, just_list=args.list)
```

Comment:
Why not put `suite` and `just_list` first like in the definition?

Reply:
They're kwargs. The order doesn't matter.